Description: Version filename of locale data (gnutls28.mo instead of
 gnutls.mo) This is necessary to make e.g. libgnutls26 and libgnutls28
 co-installable.
Author: Andreas Metzler <ametzler@debian.org>
Last-Update: 2014-05-24

--- gnutls28-3.3.2.orig/po/Makevars
+++ gnutls28-3.3.2/po/Makevars
@@ -1,7 +1,7 @@
 # Makefile variables for PO directory in any package using GNU gettext.
 
 # Usually the message domain is the same as the package name.
-DOMAIN = $(PACKAGE)
+DOMAIN = $(PACKAGE)30
 
 # These two variables depend on the location of this directory.
 subdir = po
